tests/test_page_object.py

```diff
@@ -72,7 +72,10 @@ def test_validate_url(self):
             ("http://user:pass@localhost/test", True), ("http://user:pass@localhost:8080/test", True)
         ]:
             returned_val = PageObject.validate_url(url)
-            self.assertEquals(returned_val, is_valid, msg="Url: {0}, Expected {1} but got {2}".format(url, is_valid, returned_val))
+            self.assertEquals(
+                returned_val,
+                is_valid, msg="Url: {0}, Expected {1} but got {2}".format(url, is_valid, returned_val)
```

Comment:
nit: feel free to ignore. For readability I'd put the 3rd argument (msg=...) on its own separate line too.